files/bootstrap-block-grid/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Jonathan Hornung"
+github = "https://github.com/JohnnyTheTank/bootstrap-block-grid"
+homepage = "http://plnkr.co/qkyOlC"
+description = "real block grid for Twitter Bootstrap"
+mainfile = ""
```

Comment:
tomByrer wrote:
> Honestly, I'm not sure about  +mainfile = "bootstrap3-block-grid.min.css"
I think in a few years BootStrap4 will be the standard, & you'll want to have mainfile to be be BS4, but you can't since that will break websites. **Perhaps there should be no mainfile?**